addons/hr_contract/models/hr_contract.py

```diff
@@ -232,6 +232,12 @@ def _safe_write_for_cron(self, vals, from_cron=False):
     def _assign_open_contract(self):
         for contract in self:
             contract.employee_id.sudo().write({'contract_id': contract.id})
+            contract_start_message = "Following new %s set as running today" % (contract.name)
+            if contract.job_id:
+                contract.employee_id.sudo().write({'job_id': contract.job_id})
+                contract_start_message = contract_start_message + ", the job position has been set to %s" % (contract.job_id.name)
```

Comment:
I think it would be better with punctuation like this:
```suggestion
            if contract.job_id:
                contract.employee_id.sudo().write({'job_id': contract.job_id})
                contract_start_message += ", the job position has been set to %s." % (contract.job_id.name)
            else:
                contract_start_message += "."
```